sdcm/tester.py

```diff
@@ -298,15 +298,8 @@ def __init__(self, *args):  # pylint: disable=too-many-statements
         InfoEvent('TEST_START test_id=%s' % Setup.test_id())
 
     def _init_test_timeout_thread(self) -> threading.Timer:
-        start_time = time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(self.start_time))
-
         def kill_the_test():
```

Comment:
if this function isn't killing the test anymore, maybe we should rename it ?

Reply:
Not, sure, it will kill the test at the end,  via critical event.

Reply:
keep in mind that based on @enaydanov work, it would be an option to overwrite those events level. hence this isn't directly killing the test